commercial/app/views/paidContent/card.scala.html

```diff
@@ -17,9 +17,13 @@
         </a>
     </div>
     @fragments.commercial.paidForMeta()
-    <a href="@optClickMacro@articleUrl" class="paidfor-box" data-link-name="advertisers logo">
+    <a href="<%=clickMacro%><%=articleUrl%>" class="paidfor-box paidfor-box--header" data-link-name="advertisers header">
```

Comment:
In a scala template href will still need to be `@optClickMacro@articleUrl` I think.